repository/src/main/resources/db/migration/V20201218__RDM-10041__CaseLocation.sql

```diff
@@ -0,0 +1,38 @@
+insert into field_type (created_at, reference, version, base_field_type_id)
+values (now(), 'CaseLocation', 1,
+    (select id from field_type where reference = 'Complex'
+        and jurisdiction_id is null
+        and version = (select max(version)
+        from field_type where reference = 'Complex'
+        and jurisdiction_id is null
+        and base_field_type_id is null))
+);
+
+insert into field_type (created_at, reference, version, base_field_type_id)
+values (now(), 'Region', 1,
+    (select id from field_type where reference = null
+        and jurisdiction_id is null
+        and version = (select max(version)
+        from field_type where reference = null
+        and jurisdiction_id is null
+        and base_field_type_id is null))
+);
```

Comment:
I'm sure you copied this so not your fault but I'm not sure I understand what this is doing. 
Consider that FIELD_TYPE.REFERENCE is a non null column, so I don't understand what's the point of even having a condition 
`where reference = null`

Reply:
I think we can just leave out setting base_field_type_id which will leave it as NULL, which is what we want for base types